src/cli/subcommands/car_cmd.rs

```diff
@@ -2,25 +2,22 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use std::path::PathBuf;
-use std::pin::pin;
 
-use cid::Cid;
 use clap::Subcommand;
-use futures::{AsyncRead, Stream, StreamExt, TryStreamExt};
-use fvm_ipld_car::CarHeader;
-use fvm_ipld_car::CarReader;
+use futures::{Stream, StreamExt, TryStreamExt};
 use itertools::Itertools;
+use tokio::io::{AsyncBufRead, AsyncSeek, AsyncWriteExt};
 
 use crate::ipld::CidHashSet;
-
-type BlockPair = (Cid, Vec<u8>);
+use crate::utils::db::car_stream::{Block, CarStream};
 
 #[derive(Debug, Subcommand)]
 pub enum CarCommands {
     Concat {
-        /// A list of `.car` file paths
+        /// A list of CAR file paths. A CAR file can be a plain CAR, a zstd compressed CAR
+        /// or a `ForestCAR.zst`
```

Comment:
What's a `ForestCAR.zst`? I suppose it's a `.forest.car.zst` file. Maybe using extensions supported here would be clearer.

Reply:
We can use the file extension here.

Reply:
Fixed